From 1d035856bac5d0a992eb952a01727cccccbe2696 Mon Sep 17 00:00:00 2001
From: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
Date: Sun, 1 Aug 2021 16:25:40 -0500
Subject: [PATCH 1/1] ARM:dts:aspeed: Correct register space size for JTAG
 master in AST2600
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit

Corrects register space size for JTAG master HW2 mode in AST2600.
The offests end at 0x3C, hence the size is corrected to be 0x40.

Signed-off-by: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
---
 arch/arm/boot/dts/aspeed-g6.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-g6.dtsi b/arch/arm/boot/dts/aspeed-g6.dtsi
index cac234073530..e26874f362d8 100644
--- a/arch/arm/boot/dts/aspeed-g6.dtsi
+++ b/arch/arm/boot/dts/aspeed-g6.dtsi
@@ -402,7 +402,7 @@

 			jtag0: jtag@1e6e4000 {
 				compatible = "aspeed,ast2600-jtag";
-				reg= <0x1e6e4000 0x20>;
+				reg= <0x1e6e4000 0x40>;
 				interrupts = <GIC_SPI 27 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&syscon ASPEED_CLK_AHB>;
 				resets = <&syscon ASPEED_RESET_JTAG_MASTER>;
@@ -412,7 +412,7 @@

 			jtag1: jtag@1e6e4100 {
 				compatible = "aspeed,ast2600-jtag";
-				reg= <0x1e6e4100 0x20>;
+				reg= <0x1e6e4100 0x40>;
 				interrupts = <GIC_SPI 53 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&syscon ASPEED_CLK_AHB>;
 				resets = <&syscon ASPEED_RESET_JTAG_MASTER2>;
--
2.17.1

